components/qmp6988/qmp6988.h

```diff
@@ -0,0 +1,197 @@
+
```

Comment:
here you need a standard header with a license, like in other components. You also need to add a doxygen description of the group and file - otherwise the doxygen documentation will not work